### ISSUE TITLE
Code blocks aren't working

### ISSUE THREAD

**Issue by wescpy** [gsuitedevs/md2googleslides#26]
Neither the simple example in the README...

![screenshot 2018-09-01 at 21 57 24](https://user-images.githubusercontent.com/1102504/44952341-15138a00-ae32-11e8-871b-d782216956d5.png)

... nor the more complex examples/example.md file render blocks of code in monospaced font...

![screenshot 2018-09-01 at 21 54 57](https://user-images.githubusercontent.com/1102504/44952331-c4039600-ae31-11e8-96db-e640f9ac7b96.png)

**Comment by sqrrrl** [comment]
For some reason slides API no longer accepts the chained font families (Courier New, monospace.) Changed to just use courier and will be fixed in 0.4 (will push later tonight)

**Comment by wescpy** [comment]
Thanks for this fix. I confirmed your finding by updating/testing extract_slides.js:309...

```
var style = getStyle(token, { fontFamily: 'Roboto Mono' }); // WORKS
var style = getStyle(token, { fontFamily: 'Roboto Mono,Courier New,Courier,Lucida Sans Typewriter,Lucida Typewriter,monospace' }); //FAILS
var style = getStyle(token, { fontFamily: 'Courier New' }); //WORKS
```

I'm not well-versed w/JS, but can you extract the exact API call (and perhaps what the HTTP return code [and payload] were)? I couldn't find an existing issue, so I'd like to get something I can use to submit one. I tried to find it on my own by going verbose and setting `NODE_DEBUG=cluster,net,http,fs,tls,module,timers` and re-running, but there wasn't any trace of setting font styles or full HTTP payloads in the debug output. Anything will help at this point... thanks!
